### PR TITLE
Ncsdk 1020 after release

### PR DESCRIPTION
**Pull request by ru-fu**
See the different commits.

### CODE REVIEW

File: doc/nrf/index.rst [hunk 1]
```diff
@@ -3,11 +3,12 @@
 Welcome to the |NCS|!
 #####################
 
-The |NCS| contains libraries and application examples for Nordic Semiconductor devices. It uses the |link_zephyr|_ as real-time operating system.
+The |NCS| is where you begin building low power cellular IoT applications with LTE-M and NB-IoT on the nRF9160.
+It offers a range of application samples and reference implementations, as well as a full suite of device drivers for the nRF9160 SiP.
```
Comment (b-gent) [inline doc/nrf/index.rst]: isn't it literally one driver at this point - link control?

File: doc/nrf/index.rst [hunk 2]
```diff
@@ -3,11 +3,12 @@
...
+The |NCS| includes the Zephyr™ Real Time Operating system (RTOS), which is built for connected low power products.
```
Comment (b-gent) [inline doc/nrf/index.rst]: I don't mind but Zephyr website uses just lower case and a hyphen - real-time operating system

**Comment by b-gent** [comment]
two small comments, otherwise looking good!

File: doc/nrf/index.rst (hunk 1 above)
Comment (ru-fu) [reply doc/nrf/index.rst]: This text is from Joel - I think it's okay since it's for the future anyway.

File: doc/nrf/index.rst (hunk 2 above)
Comment (ru-fu) [reply doc/nrf/index.rst]: fixed

**Pull request merged**